### PR TITLE
Fix styling

### PR DESCRIPTION
**Pull request by hoangnt2**
1. Fix styling bug of inputs
2. Replace exact match tool tip by ShowInfoOnHover component
3. Add scroll bar to LinkReleaseToProjectModal when the content is too large

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -393,24 +393,8 @@ th {
...
-    font-size: 0.875rem;
-    display: block;
-    transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
-    font-weight: 400;
-    height: 1.5rem;
-    padding-bottom: 0.25rem;
-    padding-left: 0.75rem;
-    padding-right: 0.75rem;
-    padding-top: 0.25rem;
-}
```
Comment (heliocastro) [inline src/styles/globals.css:411]: Why remove entire form.control ? This is to be exactly like liferay styling, incluDing the orange fading in.

**Review by heliocastro — COMMENTED** [review]
Some commentsLook for

File: src/styles/globals.css (hunk 1 above)
Comment (hoangnt2) [reply src/styles/globals.css:411]: @heliocastro This snippet made the input look smaller. I am not sure why we need this. I think bootstrap's default  is enough

With form-control
image

Without form-control
image

Comment (heliocastro) [reply src/styles/globals.css:411]: Ok, got the point

**Review by heliocastro — APPROVED** [review]
LGTM

**Pull request merged**